version.bzl

```diff
@@ -1,3 +1,3 @@
 # NOTE: If you change the format of this line, you must change the bash command
 # in /scripts/publish to extract the version string correctly.
-DF_VERSION = "2.6.6"
+DF_VERSION = "2.6.7"
```

Comment:
supernit: can you please bring back the trailing newline?

(i suspect otherwise some other bits might break)